libafl_qemu/librasan/asan/src/patch/raw.rs

```diff
@@ -34,108 +34,88 @@ impl Patch for RawPatch {
 
 impl RawPatch {
     #[cfg(target_arch = "x86_64")]
-    fn get_patch(_target: GuestAddr, destination: GuestAddr) -> Result<Vec<u8>, RawPatchError> {
+    fn get_patch(_target: GuestAddr, destination: GuestAddr) -> Vec<u8> {
         // mov rax, 0xdeadfacef00dd00d
         // jmp rax
-        let insns = [
-            [0x48, 0xb8, 0x0d, 0xd0, 0x0d, 0xf0, 0xce, 0xfa, 0xad, 0xde].to_vec(),
-            [0xff, 0xe0].to_vec(),
+        let addr = destination.to_ne_bytes();
+        #[rustfmt::skip]
+        let insns: &[&[u8]] = &[
```

Comment:
Random question @WorksButNotTested can't we also do dynasm here like we do in libafl_frida?
https://github.com/AFLplusplus/LibAFL/blob/d3ddc8edc4008cf7e1ca914aff84239489438589/libafl_frida/src/cmplog_rt.rs#L198

Would be more maintable than pasting random bytes, right?

Reply:
That's pretty nice actually, but dynasm doesn't support ARMv7 and PowerPC. We can migrate the supported architectures if you'd like.

Reply:
Yeah. Nobody loves powerpc! Just went with it as it was the lowest common denominator. Happy if you want to port the others though.